src/pages/UserProfile/UserProfile.jsx

```diff
@@ -54,10 +54,19 @@ const UserProfile = () => {
               />
             </Modal>
           )}
+          <img
+            className="absolute w-full h-40 object-cover -left-0 -z-10"
+            src={
+              userInfo.coverPhoto
```

Comment:
Optional chaining operator might help here.

Reply:
thanks for that will change right away